wbooks/src/app/components/BookDetailItem/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { View, Image, Text } from 'react-native';
+import CustomButton from '@app/components/CustomButton';
+import { Book } from '@interfaces/books';
+
+import styles from './styles';
+
+interface Props {
+  book: Book;
+}
+
+function BookDetailItem({ book }: Props) {
+  const { author, genre, imageUrl, title, year } = book;
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.detailContainer}>
+        <Image source={{ uri: imageUrl }} style={styles.image} />
```

Comment:
What’s happen if the `imageUrl` is null or undefined? You must use an image placeholder